tests/jquery.flot.touchNavigate.Test.js

```diff
@@ -291,8 +315,11 @@ describe("flot touch navigate plugin", function () {
                 amount = getDistance(finalCoords) / getDistance(initialCoords);
 
             simulate.sendTouchEvents(initialCoords, eventHolder, 'touchstart');
```

Comment:
These tests are already too fat. I think there should be a **sentTouchEventsAndWait()** which is calling **simulate.sendTouchEvents** and **jasmine.clock()...**. I don't like functions containing *and* but in this case I think it will be better.